packages/pn-commons/src/components/EmptyState.tsx

```diff
@@ -47,29 +49,25 @@ function EmptyState({
       }}
     >
       {FinalIcon && (
-        <FinalIcon sx={{ verticalAlign: 'middle', mr: '20px', mb: '2px', fontSize: "1.25rem" }} />
+        <FinalIcon sx={{ verticalAlign: 'middle', mr: '20px', mb: '2px', fontSize: '1.25rem' }} />
       )}
       <Typography variant="body2" sx={{ display: 'inline' }}>
         {emptyMessage}
       </Typography>
-      { emptyActionCallback && 
+      {emptyActionCallback && (
         <>
           &nbsp;
-          <Typography
-            color="primary"
-            variant="body2"
-            fontWeight={'bold'}
+          <ButtonNaked
             data-testid="callToActionFirst"
-            sx={{
-              cursor: 'pointer',
-              display: 'inline',
-            }}
             onClick={emptyActionCallback}
+            sx={{ buttonNakedInheritStyle }}
           >
-            {emptyActionLabel}
-          </Typography>
-        </> 
-      }
+            <Typography color="primary" variant="body2" fontWeight={'bold'}>
+              {emptyActionLabel}
+            </Typography>
+          </ButtonNaked>
+        </>
+      )}
       &nbsp;
       <Typography variant="body2" sx={{ display: 'inline' }}>
```

Comment:
Add secondaryMessage.emptyMessage && component. Check that this doesn't cause problem around the application.

Reply:
The same for secondaryMessage.emptyActionLabel. Check that this doesn't cause problem around the application.